storage/ndb/include/kernel/signaldata/ScanTab.hpp

```diff
@@ -149,6 +151,18 @@ class ScanTabReq {
                                         Note: these bits are ignored since
                                         7.0.34, 7.1.23, 7.2.7 and should be
                                         zero-filled until future reuse.
+ g = Aggregation           - 1(of 8) Bit 7   reuse the highest bit of Parallelism
```

Comment:
Only use 1 bit, otherwise it won't work with newer versions that makes use of the other 7 bits